CONTRIBUTING.md

```diff
@@ -57,9 +63,9 @@ Prior to opening a pull request, ensure that your code has been auto-formatted,
 and that it passes both the lint and test validation checks:
 
 ```shell
-cargo fmt  # Auto-formatting...
-cargo test  # Testing...
 cargo clippy --workspace --all-targets --all-features -- -D warnings  # Linting...
+RUFF_UPDATE_SCHEMA=1 cargo test  # Testing and updating ruff.schema.json
+pre-commit run --all-files  # rust and python formatting, markdown and python linting, etc.
```

Comment:
I always use this flag to see the actual changes made by the hooks. Do you think it's useful?

```suggestion
pre-commit run --all-files --show-diff-on-failure  # rust and python formatting, markdown and python linting, etc.
```

Reply:
this is neat, thanks